src/keyboard/Keyboard.tsx

```diff
@@ -35,8 +35,8 @@ import { useLocalStorageState } from "../misc/useLocalStorageState";
 type BehaviorMap = Record<number, GetBehaviorDetailsResponse>;
 
 function useBehaviors(): BehaviorMap {
-  let connection = useContext(ConnectionContext);
-  let lockState = useContext(LockStateContext);
+  const connection = useContext(ConnectionContext);
```

Comment:
linting changes in an otherwise untouched file, i would remove these from the PR as well